servers/rendering/shader_preprocessor.cpp

```diff
@@ -1226,6 +1271,10 @@ Error ShaderPreprocessor::preprocess(State *p_state, const String &p_code, Strin
 				state->cyclic_include_hashes.erase(code_hash); // Remove this hash.
 				return FAILED;
 			}
+			// Remove any indendentation in front of the directive
```

Comment:
This is another thing I did because of unit testing. The preprocessor replaces #defines with empty lines. I don't know if this is intentional but I didn't want to mess with that. However I found that if the #define was indented, the preprocessor would leave a line containing only a space.

Reply:
Commented this in my review.